pom.xml

```diff
@@ -258,9 +258,9 @@
     <version.org.subethamail>1.2</version.org.subethamail>
     <version.org.subethamail.subethasmtp>3.1.6</version.org.subethamail.subethasmtp>
     <version.org.webjars.swagger-ui>2.2.10</version.org.webjars.swagger-ui>
-    <version.org.wildfly>14.0.1.Final</version.org.wildfly>
-    <version.org.wildfly.arquillian>2.1.1.Final</version.org.wildfly.arquillian>
-    <version.org.wildfly.core>6.0.2.Final</version.org.wildfly.core>
+    <version.org.wildfly>18.0.1.Final</version.org.wildfly>
```

Comment:
@mareknovotny sorry, can not find better place to comment.

Do we still use `version.org.wildfly.gwt.sdm` ? I found used just in `uberfire-archetype/uberfire-project-archetype` where it is even downgraded to 11.0.

To me seems worth to remove it with this PR to facilitate future upgrades of wildfly. What is your opinion? Should be investigated/done separately?

Reply:
we unfortunately use it for showcases and GWT SDM developer environment, I was asking @adrielparedes if we can drop this special Errai WF distro now. That is work related to https://issues.redhat.com/browse/AF-2483

See original creation at https://github.com/errai/errai/commit/890b0b810f7fd269f92e2d5782a7a57293688cd1

Reply:
ok my concern was we have now `version.org.wildfly.gwt.sdm` and `version.org.jboss.errai.wildfly` for the same use case, but from your comment it seems they have different purpose

Reply:
@jomarko no, you are right I think they are used for the same purpose. My comment was more about if we don't need to use that special distro dependency, we can eliminate it completely ;)